script/docker-tests

```diff
@@ -14,13 +15,17 @@ function prepare_fullstack_test {
 
     git clone https://github.com/os-autoinst/os-autoinst.git ../os-autoinst
     cd ../os-autoinst
-    cpanm -n --mirror http://no.where/ --installdeps .
+    [ "$INSTALL_FROM_CPAN" -eq 1 ] && \
+	      (cpanm --local-lib=~/perl5 local::lib && cpanm -n --installdeps . ) || \
+	      cpanm -n --mirror http://no.where/ --installdeps .
     if ! [ $? -eq 0 ]; then
```

Comment:
in this case, if INSTALL_FROM_CPAN is not set, the exit of the cd that is previous is what's evaluated right?... I think we can remove then the check

Reply:
right, will address this in another PR